cli/printer/test.go

```diff
@@ -20,6 +20,7 @@ func (t *TestPrinter) PrintJobs(...*models.Job) error
 func (t *TestPrinter) PrintLoadBalancers(...*models.LoadBalancer) error                { return nil }
 func (t *TestPrinter) PrintLoadBalancerSummaries(...*models.LoadBalancerSummary) error { return nil }
 func (t *TestPrinter) PrintLoadBalancerHealthCheck(*models.LoadBalancer) error         { return nil }
+func (t *TestPrinter) PrintLoadBalancerIdleTimeout(*models.LoadBalancer) error         { return nil }
```

Comment:
where would this be used? i don't think we need this.

Reply:
It is used in the `l0 loadbalancer idletimeout` subcommand. See here: https://github.com/quintilesims/layer0/blob/553-jparsons-elb-timeout/cli/command/load_balancer_command.go#L419

Reply:
looks like this UX is based off of the healthcheck pattern, which isn't very good IMO. I think it would be better to base this off of something like `l0 service scale`